src/supertux/resources.cpp

```diff
@@ -92,8 +92,8 @@ Resources::get_font_for_locale(const std::string& locale)
   if(locale == "cmn" || locale == "ja" || locale == "zh_CN" || locale == "zh_TW")
     return "fonts/NotoSansCJKjp-Medium.otf";
   if(locale == "he")
-    return "fonts/Yehuda CLM Bold.ttf";
-  return "fonts/SuperTux-Medium.ttf";
+    return "fonts/shuneet3-medium.otf";
+  return "fonts/Supertux-Medium.ttf";
```

Comment:
This line has to stay here. It does not mean "set Supertux-Medium for Hebrew", it means  "set Supertux-Medium for all other languages"